### ISSUE TITLE
From caffe to pytorch: convert and val success but unable to train?

### ISSUE THREAD

**Issue by njuww** [microsoft/MMdnn#777]
Trying to convert imagenet--alexnet(caffe) to pytorch, it should be  loss≈1.9 and  top1 55%  top5  80.2% 
My  order : mmconvert -sf caffe    -iw ***   -in ***   -df pytorch  -om  ****  , semms no bug reported

when i val the model using pytorch  , i  got  loss ≈ 6.5  but  top1 54%   top5  78.4%，that  seems  strange，and  when  i  trying  to  train  the  model ，the  val  accuracy  become  worse.

the convert i made for pytorch imagenet train and val ：
1.using  std = [1,1,1]
2.convert  RGB  to  BGR
3.semms caffe do not use normalize?  so i make iput = input * 255
4.the loss-function i use is nn.CrossEntropyLoss()

i am not familiar with caffe,so  i can not find the problem.
May  anyone  give me  some  advice？
